packages/lime-app/files/96-lime-app-index_page

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+uci set uhttpd.main.index_page=lime_app_index.html
+uci commit uhttpd
+/etc/init.d/uhttpd restart
```

Comment:
restarting a service in the uci-defaults is not a good idea because at boot time this runs before uhttpd has even started to start (because boot has `START=10` and for example uhttpd runs at has `START=50`)

Reply:
Rrrrright!